operator/v1/types_console.go

```diff
@@ -39,6 +43,17 @@ type ConsoleCustomization struct {
 	// documentation URL.
 	// Invalid value will prevent a console rollout.
 	DocumentationBaseURL string `json:"documentationBaseURL,omitempty"`
+	//customProductName is the customizable product name that can be set for the web console
+	// +optional
+	CustomProductName string `json:"customProductName"`
+	//customLogoFile references a ConfigMap containing a custom product logo that can be set for the web console
+	// You only have to set this if you have your own custom product logo you wish to show on the console.
+	// The ConfigMap must exist in the openshift-config namespace and contain the following required fields:
+	// - ConfigMap.binaryData["filename"] - logo in binary form.
+	// This is best generated using: oc create configmap my-branding-config --from-file=/path/to/file -n openshift-config
+	// Image size must be less than 1 MB due to constraints on configmap size
+	// +optional
+	CustomLogoFile v1.ConfigMapNameReference `json:"customLogoFile"`
```

Comment:
This needs to be a `ConfigMapFileReference`. We need to be able to specify the key so that we have the right file extension. Otherwise the file will not be served with the right mime type.